### DISCUSSION TITLE
How to configure github actions?

### DISCUSSION THREAD

**Discussion by linrongbin16** [docsifyjs/docsify#2050] [Q&A] [answered]
Hi, how to configure github actions to deploy pages?

I don't have a `.github` folder, and github actions are not triggered, and the pages are not shown.
How should I configure `.github` workflow?

**Comment by sy-records** [comment, marked as answer]
Use GitHub pages, no need setting github actions. https://pages.github.com/

**Reply by linrongbin16** [reply]
Thanks, I found it
